livekit-plugins/livekit-plugins-deepgram/livekit/plugins/deepgram/stt.py

```diff
@@ -40,12 +47,17 @@ def __init__(
         min_silence_duration: int = 10,
     ) -> None:
         super().__init__(streaming_supported=True)
-        api_key = api_key or os.environ.get("DEEPGRAM_API_KEY")
-        if not api_key:
+        self._api_key = api_key or os.environ.get("DEEPGRAM_API_KEY")
+        if not self._api_key:
             raise ValueError("Deepgram API key is required")
 
-        dg_opts = deepgram.DeepgramClientOptions(api_key=api_key, url=api_url or "")
-        self._client = deepgram.DeepgramClient(config=dg_opts)
+        self._session = aiohttp.ClientSession(
```

Comment:
Session is leaked here. I think this should be moved into the Stream ctor

Reply:
changed to use the more canonical `async with aiohttp.ClientSession()`. No downsides to this in the stream case but added latency in the recognize case.